Project.toml

```diff
@@ -2,3 +2,6 @@ name = "GeoInterfaceRFC"
 uuid = "092d6370-f8e6-11e9-0b9b-bbd6df571de9"
 authors = ["Yeesian Ng <ngyeesian@gmail.com>"]
 version = "0.1.0"
+
+[deps]
```

Comment:
Since Test is only needed when running tests and not for the package, this should go under extras, like this:
```
[extras]
Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"

[targets]
test = ["Test"]
```

Reply:
Do we want 1.0, 1.1 compat? Otherwise we can go for a Project.toml in the tests folder.

Reply:
I think we want the 1.0 compat if we can, I wouldn't drop it for this reason.